.github/workflows/ci.yml

```diff
@@ -62,6 +62,10 @@ jobs:
           args: --all --verbose
       - name: Validate docs
         run: ./scripts/validate-docs.sh
+      - name: Setup rust-wasm target and compile test modules
+        run: |
+          rustup target add wasm32-wasi
```

Comment:
if you put this in the make target it makes, it will just skip if already present and makes it so some one runs that step they don't miss it